lib/src/Uncertainty/Algorithm/MetaModel/Kriging/openturns/GeneralizedLinearModelAlgorithm.hxx

```diff
@@ -76,7 +76,7 @@ public:
   GeneralizedLinearModelAlgorithm (const NumericalSample & inputSample,
                                    const NumericalSample & outputSample,
                                    const CovarianceModel & covarianceModel,
-                                   const BasisCollection & multivariateBasis,
+                                   const BasisCollection & basisCollection,
                                    const Bool normalize = ResourceMap::GetAsBool("GeneralizedLinearModelAlgorithm-NormalizeData"),
                                    const Bool keepCholeskyFactor = ResourceMap::GetAsBool("GeneralizedLinearModelAlgorithm-KeepCovariance"));
 
```

Comment:
With your changes, the KeepCholeskyFactor is obsolete.
It should be removed in ctor